spec/features/single_content_item_spec.rb

```diff
@@ -234,20 +234,16 @@
         expect(page).to have_selector('.govuk-link', text: I18n.t("metrics.searches.external_link"))
       end
 
-      it 'renders the metric timeseries for satisfaction' do
-        satisfaction_rows = extract_table_content(".chart.satisfaction table")
-        expect(satisfaction_rows).to match_array([
-          expected_table_dates,
-          ["User satisfaction score", "100%", "90%", "80%"]
-        ])
+      it 'renders the satisfaction score' do
+        expect(page).to have_selector('.metric-summary__satisfaction', text: '')
```

Comment:
Out of interest, what's the `text: ''` bit doing?

Reply:
Sorry that was a mistake. But it also made me realise that this test wasn't testing the right thing anyway as it had almost exactly the same selector as earlier on the file.

Reply:
No need to apologise, I can see it's fixed now :+1: